pyk/src/tests/unit/utils.py

```diff
@@ -16,3 +18,19 @@
 f, g, h = map(KLabel, ('f', 'g', 'h'))
 
 k = KLabel('<k>')
+
+
+def lt(var: str, n: int) -> KApply:
+    return mlEqualsTrue(ltInt(KVariable(var), intToken(n)))
+
+
+def ge(var: str, n: int) -> KApply:
+    return mlEqualsTrue(geInt(KVariable(var), intToken(n)))
+
+
+def config(var: str) -> KApply:
+    return KApply('<top>', KVariable(var))
+
+
+def config_int(n: int) -> KApply:
+    return KApply('<top>', intToken(n))
```

Comment:
Provide some sugars.

Reply:
Some of these (or something similar) is already available in: https://github.com/runtimeverification/k/blob/847da52e27d54434df0d1dad89146c8d0e371840/pyk/src/pyk/prelude/kint.py

Reply:
I think I used some functions in kint to make my test cases simpler. If I use kint directly, test cases will be complex.

Reply:
You're right, I didn't notice you're already using these functions inside the helpers.